src/System.Private.CoreLib/ILLinkTrim.xml

```diff
@@ -40,6 +40,7 @@
     <!-- Accessed via reflection in TraceLogging-style EventSource events. -->
     <type fullname="*f__AnonymousType*" />
     <!-- Accessed via native code. -->
+    <type fullname="System.Runtime.InteropServices.ComTypes.IEnumerable" />
```

Comment:
Do you really need it here? There are other similar interfaces, like `IExpando`, that are rooted implicitly by other means.

Reply:
This type needs to be rooted somewhere otherwise it gets linked out. I'm currently looking to see if there's a better place to root it.

Related: we should extend the `CreateRuntimeRootILLinkDescriptorFile` task to also parse the src/vm/mngstditflist.h file since that's where this type (along with the types for other custom marshalers that we haven't added since their types aren't exposed from System.Private.CoreLib) are rooted.

Reply:
Ok. 

> we should extend the CreateRuntimeRootILLinkDescriptorFile task

It is probably not worth the added complexity. The list in `mngstditflist.h` is very unlikely to change, so optimizing for changing it is not worth it.

Reply:
So I could add these types to `mscorlib.h`, but they'd only be used by the `CreateRuntimeRootILLinkDescriptorFile` task to ensure they aren't linked away. The definitions in `mscorlib.h` would never be used by the runtime with the current code.

Reply:
I think the way you have it right now is fine.